### DISCUSSION TITLE
How to update from v45 to 48?

### DISCUSSION THREAD

**Discussion by sergey-novozhylov** [expo/expo#21930] [Archived: Questions]
Hi guys, 
thank you for the project you did.
i read an article about the new version
but there is only information "how to update from v47 to 48"

What are my steps if my current version is 45?
should i update step-by-step to 46-47-48?
or is there a chance to proceed the one update 45-48?
Regards,
Sergey

**Comment by suhaybkaofficial** [comment]
npx expo update
